File: app/src/main/java/com/itba/runningMate/running/fragments/map/RunningMapView.java
package com.itba.runningMate.running.fragments.map;

import com.itba.runningMate.mainpage.fragments.running.model.Route;


public interface RunningMapView {

    void attachTrackingService();

    void detachTrackingService();

    void showRoute(Route route);

    void showDefaultLocation();

    void showLocation(double latitude, double longitude);

}

File: app/src/main/java/com/itba/runningMate/running/fragments/map/RunningMapView.kt
package com.itba.runningMate.running.fragments.map

import com.itba.runningMate.domain.Route

interface RunningMapView {

    fun attachTrackingService()

    fun detachTrackingService()

    fun showRoute(route: Route)

    fun showDefaultLocation()

    fun showLocation(latitude: Double, longitude: Double)

}